analytics/policy/tests/unit/test_business_licence_client.py

```diff
@@ -0,0 +1,48 @@
+import json
+from unittest.mock import Mock, patch
+
+import pytest
+from policy.services.business_licence_client import BusinessLicenceClient
+
+
+@pytest.fixture
+def client():
+    return BusinessLicenceClient()
+
+
+def mock_response(response_data=None):
+    mock_response = Mock()
+    if response_data:
+        mock_response.data = json.dumps(response_data).encode("utf-8")
+    else:
+        mock_response.data = None
+    return mock_response
+
+
+@patch("policy.services.business_licence_client.urllib3.PoolManager")
+@pytest.mark.parametrize(
+    "licence_number, expected_status, expected_count",
+    [
+        ("0", [], 0),
```

Comment:
try 
None,
"", 
and see what the output would be.

Reply:
also is there a reason why number of listing is an output here? do you expect the output to change in the future?

Reply:
> try None, "", and see what the output would be.

I added them and the output would still be empty list []
![image](https://github.com/OpenBCca/airbnb-regulation/assets/32988140/98184cd7-874f-418e-bb77-c55616ed3a06)